webpack/assets/javascripts/react_app/components/common/DateTimePicker/DatePicker.js

```diff
@@ -103,7 +90,7 @@ DatePicker.propTypes = {
   locale: PropTypes.string,
   weekStartsOn: PropTypes.number,
   id: PropTypes.string,
-  placement: OverlayTrigger.propTypes.placement,
+  placement: PropTypes.string,
```

Comment:
can you rename it to `position`?

Reply:
That might be a breaking change so I'll leave it